demos/serverless/README.md

```diff
@@ -61,6 +61,18 @@ npm run deploy -- -r us-east-1 -b <deploy-bucekt> -o <capture-bucket-prefix> -i
 
 Note that you need to enable these regions if you plan to use media capture. For more information, see [Managing AWS Regions](https://docs.aws.amazon.com/general/latest/gr/rande-manage.html).
 
+##### ChimeSDKMediaPipelines Namespace vs Chime Namespace
+The AWS.Chime and the AWS.ChimeSDKMediaPipelines are both Amazon Chime's AWS clients to help builders create Amazon Chime SDK media pipelines. AWS.ChimeSDKMediaPipelines is intended to replace the previous AWS.Chime client.
```

Comment:
nit: put namespaces into a code block will help, e.g., `AWS.Chime` and `AWS.ChimeSDKMediaPipelines`.

Reply:
@lognoel Looks like this is not addressed.